src/cocaine/burlak/config.py

```diff
@@ -157,14 +157,16 @@ class Config(object):
                     'type': 'string',
                     'allowed': [
                         'tvm',
+                        'tvm2',
                         'TVM',
+                        'TVM2',
                         'promiscuous',
                         'test1', 'test2'
                     ]
                 },
                 'client_id': {'type': 'integer'},
                 'client_secret': {'type': 'string'},
-                'tok_update': {'type': 'integer', 'required': False},
+                'token_expiration_s': {'type': 'integer', 'required': False},
```

Comment:
Expiration time could be limited by scheme to be positive values only. It could be also some reasonable upper limit.